File: Count Elements With Maximum Frequency.java
import java.util.Arrays;

class Solution {
    public int maxFrequencyElements(int[] nums) {
        
        int[] hash = new int[101];
        int n = nums.length;
        
        for (int i = 0; i < n; i++) {
            hash[nums[i]]++;
        }
        
        int maxi = Arrays.stream(hash).max().getAsInt();
        int ans = 0;
        
        for (int i = 0; i < 101; i++) {
            if (hash[i] == maxi) {
                ans += hash[i];
            }
        }
        return ans;
    }
}

File: Count Elements With Maximum Frequency.py
class Solution:
    def maxFrequencyElements(self, nums):
        hash = [0] * 101
        n = len(nums)
        
        for i in range(n):
            hash[nums[i]] += 1
        
        maxi = max(hash)
        ans = 0
        
        for i in range(101):
            if hash[i] == maxi:
                ans += hash[i]
        
        return ans

